contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -99,6 +99,32 @@ var UserModel = BaseModel.extend({
     },
     get_full_name: function(){
     	return this.get('first_name') + " " + this.get('last_name');
+    },
+    get_channels: function(){
+    	var self = this;
+	    return new Promise(function(resolve, reject){
+	        $.ajax({
+	            method:"GET",
+	            url: window.Urls.get_user_channels(),
+	            error: reject,
+	            success: function(data) {
+	            	resolve(new ChannelCollection(JSON.parse(data)));
+	            }
+	        });
+	    });
+    },
+    get_pending_invites: function(){
+    	var self = this;
+	    return new Promise(function(resolve, reject){
```

Comment:
Whoa. I wasn't expecting to see promises. This is an ES6 feature, and I'm not seeing a shim for it in older browsers. They seem to be pretty integrated into the frontend. Was this intentional? What's our target browser version? Why are we holding back from using other ES6 features?